torch/autograd/grad_mode.py

```diff
@@ -60,11 +61,16 @@ class no_grad(_DecoratorContextManager):
         >>> z.requires_grad
         False
     """
+    def __init__(self):
+        if not torch._jit_internal.is_scripting():
+            super().__init__()
+        self.prev = False
+
     def __enter__(self):
         self.prev = torch.is_grad_enabled()
-        torch._C.set_grad_enabled(False)
+        torch.set_grad_enabled(False)
 
-    def __exit__(self, *args):
+    def __exit__(self, exc: Any, type: Any, tb: Any):
```

Comment:
probably this doesn't matter, but it's documented as taking `exc_type, exc_value, traceback`.

Reply:
Good catch, I changed this.